blockchains/smartchain/assets/0xc53708664b99df348dd27c3ac0759d2da9c40462/info.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Gourmetgalaxy",
+    "website": "https://gourmetgalaxy.io/",
+    "source_code": "https://github.com/gourmetgalaxy",
+    "description": "Gourmet Galaxy is a fusion between a DeFi Yield Farming, a NFTs Gaming dApp, LaunchPad platform aim at solving the problems that overshadowed Yield Farming platforms.",
+    "socials": [
+        {
+            "name": "Twitter",
+            "url": "https://twitter.com/gourmetfinance",

```

Comment:
I suggest removing the Twitter account until the suspension is removed. This has happened to multiple accounts already and I am not sure when Twitter plans to fix this.